src/VisualStudio/CSharp/Repl/CSharpInteractiveCommandHandler.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.CSharp.Interactive
 {
-    [ExportCommandHandler("Interactive Command Handler", ContentTypeNames.CSharpContentType)]
+    [ExportCommandHandler("Interactive Command Handler")]
```

Comment:
How does one invoke those?  I don't see them in the context menu (even before my change).